docs/ingestion/input-sources.md

```diff
@@ -581,9 +584,11 @@ in `druid.ingestion.hdfs.allowedProtocols`. See [HDFS input source security conf
 The HTTP input source is to support reading files directly from remote sites via HTTP.
 
 :::info
```

Comment:
```suggestion
:::info Security notes
```
This becomes the [title](https://docusaurus.io/docs/2.x/markdown-features/admonitions#specifying-title) of the admonition